### PR TITLE
Enhancing contrast and brightness of an image #88

### LINKED ISSUE #88: Implement Enhance brightness for images as apart of Augmentations
Implement a class to enhance image brightness  of a image

### PR DESCRIPTION
**Pull request by prakharshreyash15**
# Description:
I have implemented a class to enhance image brightness and contrast of a image using Python

Fixes #88 

## Type of change:

-[X]Added code to implement a class to enhance the contrast and brightness of an image.

 
# Checklist:
- [X ] My code follows the style guidelines of this project.
- [ X] I have performed a self-review of my own code.
- [X ] I have commented my code, particularly in hard-to-understand areas.
- [ X] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.

### CODE REVIEW

**Comment by chebroluharika** [comment]
@prakharshreyash15 please include this change in dataAugmentations.py file as one module and also attache screenshots for this change.

File: Implement a class to enhance image brightness and contrast of a image.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+def controller(img, brightness=255, contrast=127):
+	
+	brightness = int((brightness - 0) * (255 - (-255)) / (510 - 0) + (-255))
+
+	contrast = int((contrast - 0) * (127 - (-127)) / (254 - 0) + (-127))
+
+	if brightness != 0:
+
+		if brightness > 0:
```
Comment (chebroluharika) [inline Implement a class to enhance image brightness and contrast of a image.py:24]: Please follow PEP8 standards for coding spaces.

**Comment by yvkrishna** [comment]
@prakharshreyash15, can you attach the sample images

**Comment by yvkrishna** [comment]
> @prakharshreyash15 please include this change in dataAugmentations.py file as one module and also attache screenshots for this change.

Yes,

But to change brightness we can use Log Transformation or Power-Law Transformation.

They are easy to use and we can provide the gain as input to the user.

What do you say @chebroluharika

**Comment by chebroluharika** [comment]
> > @prakharshreyash15 please include this change in dataAugmentations.py file as one module and also attache screenshots for this change.
> 
> Yes,
> 
> But to change brightness we can use Log Transformation or Power-Law Transformation.
> 
> They are easy to use and we can provide the gain as input to the user.
> 
> What do you say @chebroluharika

@yvkrishna any technique would be fine to me. Ultimately image should get brighten and contrasted according to given conditions given by user.

**Comment by yvkrishna** [comment]
@prakharshreyash15, look at data augumentation python file and according to that code format, try to make changes.

There are other methods as well in that class. So check them and make possible changes.

**Comment by yvkrishna** [comment]
> > > @prakharshreyash15 please include this change in dataAugmentations.py file as one module and also attache screenshots for this change.
> > 
> > 
> > Yes,
> > But to change brightness we can use Log Transformation or Power-Law Transformation.
> > They are easy to use and we can provide the gain as input to the user.
> > What do you say @chebroluharika
> 
> @yvkrishna any technique would be fine to me. Ultimately image should get brighten and contrasted according to given conditions given by user.

Okay, those methods consume less code that's why I have suggested that. 😅

**Comment by chebroluharika** [comment]
@prakharshreyash15 do we have any update on this. Deadline is approaching!!

**Comment by ainy-123** [comment]
> @prakharshreyash15 do we have any update on this. Deadline is approaching!!

Hey @chebroluharika ,can I work on this issue?
in case, @prakharshreyash15 is not working on it.

**Comment by chebroluharika** [comment]
Closing this as we don’t have update since long time.

**Pull request closed without merging**